docs/modules/ROOT/pages/how-tos/gke_node_maintenance.adoc

```diff
@@ -0,0 +1,36 @@
+= GKE Node Maintenance
```

Comment:
For someone not familiar with GKE, this is more confusing than helpful. At least I do not get it.

I understand, that raising the Kubernetes Version is a intentional change done within the Terraform repository of that cluster. Will this have an imediate effect? The Auto-Upgrade suggests no. But then I am supposed to log into the Google Console so I can act when things go thouth. This suggests that I have to babysit the process.

So my assumption is this:

I change the Kubernetes Version in code. Then I ensure the terraform apply got executed (pipeline). And herer my job is done.

The actuall upgrade will happen automatically within the next maintenance window.

If things go south, monitoring should alert. Here comes the need for logging into the Google Console. So mentioning this would go more into a troubleshoot section which could be a how to on its own.

Reply:
thx for the feedback. I'll try to rewrite it and make it more clear.

Reply:
Yes thanks. This now makes sense to me.